iradix_test.go

```diff
@@ -43,11 +43,44 @@ func CopyLeaf(l *leafNode) *leafNode {
 	return ll
 }
 
+func TestRadix_HugeTxn(t *testing.T) {
+	r := New()
+
+	// Insert way more nodes than the cache can fit
```

Comment:
Any thoughts about maybe running through some of the same UUIDs later to exercise the case where a node is modified during a transaction, but it's not in the cache any more?